tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Differ;
+
+use PhpCsFixer\Differ\SebastianBergmannDiffer;
+use PHPUnit\Framework;
+use Prophecy\Argument;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Differ\SebastianBergmannDiffer
+ */
+final class SebastianBergmannDifferTest extends Framework\TestCase
+{
+    public function testIsDiffer()
+    {
+        $differ = new SebastianBergmannDiffer();
+
+        $this->assertInstanceOf('PhpCsFixer\Differ\DifferInterface', $differ);
+    }
+
+    public function testDiffReturnsDiff()
```

Comment:
not sure here.

it looks like here you are not testing the functionallity of differ, but it's implementation details.

why not to use that real Differ instead of mocked one and compare inputs against real, expected diff blob ?

currently, this test check if external SB Differ is called and proxied by our differ adapter class.
if one would replace external implementation with different one that provides different format, this test would still work after.

Instead of testing "is this class properly do the proxying", I would test "Is the public output for public params is matching expected one", regardless how it's generated under the hood.